config/initializers/default_url_options.rb

```diff
@@ -0,0 +1,13 @@
+default_domains = {
```

Comment:
This change is great 👍 Clear to understand how it works across all environments and easy to make changes with confidence whilst keeping high confidence that the fallback hosts will still work.